content/microcontrollers/_index.md

```diff
@@ -8,7 +8,7 @@ weight: 3
 
 TinyGo lets you run Go directly on microcontrollers.
 
-TinyGo has support for 44 different boards and devices such as the Arduino Nano33 IoT, Adafruit Circuit Playground Express, BBC micro:bit and more. Click on a board name found in the left menu to see the what features are supported for the given hardware.
+TinyGo has support for 46 different boards and devices such as the Arduino Nano33 IoT, Adafruit Circuit Playground Express, BBC micro:bit and more. Click on a board name found in the left menu to see the what features are supported for the given hardware.
```

Comment:
I think `48 boards` instead of `46 boards`.

```
sago35@sago35-B450GT3  ~/dev/src/github.com/tinygo-org/tinygo-site/content/microcontrollers (prepare-0.16.0-release)
$ ls -1 *.md | grep -v _index.md
arduino-mega2560.md
arduino-nano.md
arduino-nano33-iot.md
arduino-uno.md
arduino-zero.md
bbc-microbit.md
bluepill.md
circuit-playground-bluefruit.md
circuit-playground-express.md
clue-alpha.md
digispark.md
esp32-coreboard-v2.md
esp32-mini32.md
esp8266-d1mini.md
esp8266-nodemcu.md
feather-m0.md
feather-m4.md
feather-nrf52840.md
feather-stm32f405.md
gameboy-advance.md
hifive1b.md
itsybitsy-m0.md
itsybitsy-m4.md
itsybitsy-nrf52840.md
maixbit.md
metro-m4-airlift.md
nintendo-switch.md
nrf52840-mdk.md
nucleo-f103rb.md
particle-argon.md
particle-boron.md
particle-xenon.md
pca10031.md
pca10040.md
pca10056.md
pinetime.md
pybadge.md
pygamer.md
pyportal.md
qtpy.md
reelboard.md
stm32f4discovery.md
teensy36.md
teensy40.md
trinket-m0.md
wioterminal.md
x9-pro.md
xiao.md
```

Reply:
Count updated.